### PR TITLE
Added Basic SignUp  Page

### PR DESCRIPTION
**Pull request by TMatome**

### CODE REVIEW

File: src/js/components/Home/Home.js [hunk 1]
```diff
@@ -20,13 +20,19 @@ class Home extends Component {
...
-      <h4> Insert speedy picture here </h4>
-      <Mapbuttons />
-      {/*<ArticleApp/>*/}
+        <NavLink to="/Books">
+          {" "}
+          <button className="btn btn-success"> Books </button>{" "}
+        </NavLink>
+        <h4> Insert speedy picture here </h4>
+        <Mapbuttons />
+        {/*<ArticleApp/>*/}
```
Comment (clewarne) [inline src/js/components/Home/Home.js]: Please remove commented out code if you do not need it. If you plan on using it in the future add a TODO comment with short explanation why

File: src/js/components/LoginPage/Login.sass [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.Login
+.Login form
```
Comment (clewarne) [inline src/js/components/LoginPage/Login.sass]: please choose a naming convention on your css classes and stick with it. Most commonly used is either camelCase or dash-separated

File: src/js/components/LoginPage/Login.js [hunk 3]
```diff
@@ -86,12 +87,20 @@ class Login extends Component {
   };
 
   render() {
+    var loc_navBarTitle = "KnowYourNation";
```
Comment (clewarne) [inline src/js/components/LoginPage/Login.js]: please stick with either camelCase OR under_score naming conventions. don't mix the two

**Review by clewarne — CHANGES_REQUESTED** [review]
Just wanna say ;) thanks for consistent formatting!

File: src/js/components/LoginPage/ButtonStyles.css [hunk 4]
```diff
@@ -1,9 +1,10 @@
 .btn-circle {
```
Comment (clewarne) [inline src/js/components/LoginPage/ButtonStyles.css]: you have .sass files and .css files? Please choose one styling method

**Comment by TMatome** [comment]
Thanks a lot,I appreciate your comments.
I will make it better

File: src/js/components/Articles/ArticleReader.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      <div style={{ position: "relative", height: "100%" }}>
+        <Navbar titleFromParent={navBarTitle} navbarItems={navbarItems} />
+        {/* The ReactReader will expand to 100% of width/height, so be sure to set a
+         height on the parent element, either with position it absolute of window,
+         set height or use paddingTop for proporsjonal scaling */}
+
+        <ReactReader
+          url={"https://gerhardsletten.github.io/react-reader/files/alice.epub"}
+          location={"epubcfi(/6/2[cover]!/6)"}
+          locationChanged={epubcifi => console.log(epubcifi)}
```
Comment (clewarne) [inline src/js/components/Articles/ArticleReader.js]: Careful of console.logs! Please always remove them before pushing your code

File: src/js/components/Articles/ArticleReader.sass [hunk 6]
```diff
@@ -0,0 +1,7 @@
+
+.Btn
+    margin: 0 auto
+    margin-top: 570px
```
Comment (clewarne) [inline src/js/components/Articles/ArticleReader.sass]: This is a very large top margin. What are you trying to achieve with it?

**Review by clewarne — APPROVED** [review]

File: src/js/components/Articles/ArticleReader.js (hunk 5 above)
Comment (TMatome) [reply src/js/components/Articles/ArticleReader.js]: Alright I will do,thanks

File: src/js/components/Articles/ArticleReader.sass (hunk 6 above)
Comment (TMatome) [reply src/js/components/Articles/ArticleReader.sass]: i was trying to put the buttons in the bottom of the page,truly speaking I sucks with styling but i will learn scss

Comment (TMatome) [reply src/js/components/Articles/ArticleReader.sass]: I have new version of this repo with your comments and recommendation,thanks a lot

File: src/js/components/Articles/ArticleReader.js [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+  Logo,
+  CloseButton,
+  CloseIcon,
+  FontSizeButton
+} from "./Components";
+import Button from "react-bootstrap/Button";
+
+const storage = global.localStorage || null;
+
+const GlobalStyle = createGlobalStyle`
```
Comment (clewarne) [inline src/js/components/Articles/ArticleReader.js:24]: I don't know what you are doing here, but this should be in a stylesheet not a component. (as in a .scss file or a .sass file or a .css file)

File: src/js/components/Articles/modules/EpubView/style.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+export default {
```
Comment (clewarne) [inline src/js/components/Articles/modules/EpubView/style.js:1]: is this supposed to stylesheet?

**Review by clewarne — CHANGES_REQUESTED** [review]
I'm not sure what is going on here but your styling needs to be addressed. You have some styles in .sass files and some in .js files. Keep all your classes in SASS files.

File: src/js/components/Articles/modules/EpubView/style.js (hunk 8 above)
Comment (TMatome) [reply src/js/components/Articles/modules/EpubView/style.js:1]: no its the layout of the ereader container

File: src/js/components/Articles/ArticleReader.js (hunk 7 above)
Comment (TMatome) [reply src/js/components/Articles/ArticleReader.js:24]: Ooh I see I got the code like that,I will try to move them out without breaking anything.

**Pull request closed without merging**